diff --git a/Cargo.lock b/Cargo.lock
index cd8f32b..7fe0fee 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -2584,7 +2584,6 @@ dependencies = [
  "embassy-rp",
  "embassy-sync",
  "embassy-time",
- "embedded-io",
  "json",
  "panic-probe",
  "portable-atomic",
diff --git a/Cargo.toml b/Cargo.toml
index 4e377e6..84f16e0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,7 +15,6 @@ embassy-sync = { version = "0.7", features = ["defmt"] }
 trouble-host = { version = "0.5.1", features = ["gatt", "scan", "derive", "security"] }
 rmk = { path = "./rmk/rmk", features = ["pico_w_ble", "async_matrix", "split", "storage"] }
 embassy-time = { version = "0.5", features = ["defmt"] }
-embedded-io = "0.6"
 # RP2350 (Pico 2 W) 対応のため embassy-rp 0.9 系に揃える
 embassy-rp = { version = "0.9", features = [
     "rp235xa",
